tools/serverpod_cli/lib/src/generator/open_api/open_api_extensions.dart

```diff
@@ -0,0 +1,68 @@
+// ignore_for_file: unused_element
+
+part of openapi_definition;
+
+extension on String {
+  /// convert queryFrom -> query-form
+  String get camelToKebabCase {
+    if (!contains(RegExp(r'[A-Z]'))) {
+      return this;
+    }
+    return replaceAllMapped(RegExp(r'([a-z])([A-Z])'),
+        (match) => '${match.group(1)}-${match.group(2)?.toLowerCase()}');
+  }
+}
+
+/// example
+/// convert [nullType] -> [null]
+///
+extension on SchemaObjectType {
+  String get type => name.replaceAll('Type', '');
+}
+
+/// example
+/// convert [dateTime] -> [date-time]
+/// convert [string] -> [string]
+///
+extension ChangeFormat on SchemaObjectFormat {
+  String get format => name.camelToKebabCase;
```

Comment:
It returns not a format but a `formattedName`, correct?

Reply:
yes